app/helpers/social_helper.rb

```diff
@@ -13,4 +13,9 @@ def answer_share_url(answer)
       protocol: (APP_CONFIG["https"] ? :https : :http),
     )
   end
+
+  def answer_copy_content(answer)
+    # copy content template: "question - answer [link]"
+    "#{answer.question.content} - #{answer.content} [#{answer_share_url(answer)}]"
+  end
```

Comment:
You can just use the `SocialHelper::TwitterMethods#prepare_tweet` method, which returns exactly the same format.

The [] around the link in the issue were also just for demonstration, they are not required in the actual output ;)